android/gradle.properties

```diff
@@ -31,4 +31,6 @@ org.gradle.daemon=false
 org.gradle.jvmargs=-XX\:MaxHeapSize\=4608m -Xmx4608m
 
 MYAPP_UPLOAD_STORE_FILE=bitcoin-jungle-prod.keystore
-MYAPP_UPLOAD_KEY_ALIAS=bitcoin-jungle-prod
\ No newline at end of file
+MYAPP_UPLOAD_KEY_ALIAS=bitcoin-jungle-prod
+MYAPP_UPLOAD_STORE_PASSWORD=YOUR_PASSWORD_HERE
+MYAPP_UPLOAD_KEY_PASSWORD=YOUR_PASSWORD_HERE
```

Comment:
I'm guessing this is not used to build production, but if it is, I'm sure this will cause problems ☠️

Reply:
Yes, these are used to build prod. We want to store `MYAPP_UPLOAD_STORE_PASSWORD` and `MYAPP_UPLOAD_KEY_PASSWORD` outside of the repo in `~/.gradle/gradle.properties`. There's some hierarchy magic here where it'll use these values from one's home folder when building for prod inside the project. I think `MYAPP_UPLOAD_KEY_ALIAS` is already in that file on our `main` branch by default?

I would change the `README` instructions to tell the user to do ^^ in their `~` folder. They just have to be set, and don't need the actual keys to build for dev env.